data/campaigns/atl02.wmf/player_names

```diff
@@ -0,0 +1,22 @@
+# Automatically created by Widelands 1.0~git24923[de6c28e@static_link] (Release)
+
+[global]
+packet_version="2"
+
+[player_1]
+name="Scenario player"
```

Comment:
How about Jundlina?

Reply:
well (spoiler) when I am finished Maybe Jundlina is gone